docs/mkdocs.yml

```diff
@@ -11,7 +11,7 @@ theme:
     name: material
     custom_dir: overrides
     logo: stylesheets/Logo_Transparent.png
-    favicon: stylesheets/spades.svg
+    favicon: stylesheets/favicon.png
```

Comment:
Adding this file should coincide with deleting `spades.svg`

Reply:
Just deleted, good call.